f8a_worker/storages/s3_manual_tagging.py

```diff
@@ -0,0 +1,24 @@
+from . import AmazonS3
+
+
+class S3ManualTagging(AmazonS3):
+    def store_user_data(self, input_json):
+        file_name = self.get_file_name(input_json.get('user'))
+        data = input_json.get('data', {})
+        ecosystem = data.get('ecosystem', None)
+        file_name = ecosystem + "/" + file_name
```

Comment:
This will again fail:

```python
>>> ecosystem = None
>>> ecosystem + "foo"
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'

```

Reply:
The same happens when there's no `'user'` key in `input_json`.